packages/as-sha256/package.json

```diff
@@ -30,26 +30,21 @@
   "scripts": {
     "clean": "rm -rf ./dist && node -r ts-node/register ./scripts/codegen.ts",
     "build": "yarn asbuild:untouched && yarn asbuild:optimized && yarn build:lib",
-    "asbuild:untouched": "asc assembly/index.ts -b build/untouched.wasm -t build/untouched.wat --runtime none --validate --debug",
-    "asbuild:optimized": "asc assembly/index.ts -b build/optimized.wasm -t build/optimized.wat --runtime none --validate -O3z --noAssert",
+    "asbuild:untouched": "asc assembly/index.ts -b build/untouched.wasm -t build/untouched.wat --runtime minimal --target debug",
+    "asbuild:optimized": "asc assembly/index.ts -b build/optimized.wasm -t build/optimized.wat --runtime minimal --target release --noAssert",
     "build:lib": "run -T tsc -p tsconfig.build.json",
     "build:web": "run -T webpack --mode production --entry ./index.js --output ./dist/as-sha256.min.js",
-    "test": "yarn run test:as-ci && yarn run test:unit",
+    "test": "yarn run test:unit",
     "test:unit": "yarn run test:unit:node && yarn run test:unit:browser",
     "test:unit:node": "run -T mocha -r ts-node/register test/unit/*.test.ts",
     "test:unit:browser": "run -T karma start --single-run --browsers ChromeHeadless,FirefoxHeadless karma.config.js",
     "benchmark": "node -r ts-node/register ./node_modules/.bin/benchmark 'test/perf/index.test.ts'",
     "benchmark:local": "yarn benchmark --local",
-    "test:ci": "yarn test:as-ci",
-    "test:as": "asp --nortrace --verbose",
-    "test:as-ci": "asp --nortrace 2> /dev/null"
+    "test:ci": "yarn test:as-ci"
```

Comment:
`test:as-ci` has been removed?

Reply:
there is no `as` tests for now and that causes tests not to be scanned by CI for a while due to this. Removing it helps run tests again in CI

Reply:
Wouldn't calling `yarn test:as-ci` fail then? As `test:ci` is calling it?
`test:ci` doesn't appear to be used (only `test`) so maybe it could be removed too?